build/logging-processor/src/main/java/org/infinispan/logging/processor/report/XmlReportWriter.java

```diff
@@ -54,7 +54,7 @@ public void writeDetail(final Element element) throws XMLStreamException {
          writeCharacters("id", prjCode.toString());
          writeCharacters("message", message.value());
          writeCharacters("description", description.value());
-         writeCharacters("level", logMessage.level().name());
+         writeCharacters("level", logMessage == null ? "EXCEPTION" : logMessage.level().name());
```

Comment:
@oraNod can you elaborate:
```
I inserted a description for a message without an ID but that was not printed to the log.html report.
```
All log messages/exceptions have a unique id.

Reply:
> Is it the case that this message inherits `@LogMessage(level = WARN)` at line 875 during execution?

No. This Message is an exception, not a log message, so it doesn't have a level associated with it.





> If all of that is true then should we also add `@LogMessage()` when we add `@Description` so that they get printed to logs.html?

`@LogMessage()` is only required for log statements. The easy way to differentiate between a log and an exception is to look at the return type of the signature. If it's `void` then it's a log message, if something is returned then it's an exception as the exception is what's returned so Infinispan can throw it.

Reply:
@ryanemerson Thanks for the great explanation! Obviously I didn't see that before so now the changes in the PR make a lot more sense. Apologies if this is just basic and I'm adding a bit of churn. Knowing this will help when getting into the work with the logging descriptions for sure though.